live_tests/test_lan_live.py

```diff
@@ -132,10 +131,8 @@ def test_get_lan_members(self):
             lan_id=self.lan['id'])
 
         self.assertGreater(len(members), 0)
-        assertRegex(self, members['items'][0]['id'], self.resource['uuid_match'])
         self.assertEqual(members['items'][0]['type'], 'nic')
         self.assertEqual(members['items'][0]['properties']['name'], self.resource['nic']['name'])
-        assertRegex(self, members['items'][0]['properties']['mac'], self.resource['mac_match'])
```

Comment:
Fixed it to work with both python 2.7 and 3.0 
Minimum requirement for `assertRegex` to function is six 1.10
I've added that change to `requirements.txt`